### ISSUE TITLE
Filling method error

### ISSUE THREAD

**Issue by Iamat66** [inkstitch/inkstitch#1778]
Hi,
When I use : Contour Fill , Single spiral, Round
I have this problem, the end of the contour fill is not correct (see picture ans .SVG file)
![13C4F538-99FA-4FAF-9B20-1B71781E30E6](https://user-images.githubusercontent.com/110776112/183702637-a74a03ef-9df4-4d95-8c6e-83fbe06fefbc.jpeg)
![RP3 Mistake](https://user-images.githubusercontent.com/110776112/183703587-627c3fb9-601e-455a-931e-2751f48ba171.svg)

**Comment by Iamat66** [comment]
@kaalleen 
I have tested with this release (Windows) , but it don't solve the problem

https://github.com/inkstitch/inkstitch/releases/tag/dev-build-kaalleen-thisandthat01

**Comment by kaalleen** [comment]
Yes, that's right. It doesn't change the single spiral algorithm. With your shape it has no way to go when it narrows down. So it's doing these strange things. Which isn't optimal.

If you can live with 0.1 mm difference, set the spacing between rows to 0.3 mm. This may help.
